### DISCUSSION TITLE
how to use HMR at vue-cli with webpack4.x?

### DISCUSSION THREAD

**Discussion by FeiFanLiang** [vuejs/pinia#1072] [Help and Questions]
webpack4.x not support import.meta,and doc have not how use HMR in webpack project,hope to add example

**Comment by gurvancampion** [comment]
I'm also trying to figure out how to use it in a project using Nuxt 2 & Webpack 4.
I tried to use `module.hot` from webpack 4 https://v4.webpack.js.org/api/hot-module-replacement/#module-api :
```js
if (module.hot) module.hot.accept(acceptHMRUpdate(useStore, module.hot))
```

But it doesn't work, I guess there may be some adaptations to do in `acceptHMRUpdate`, I don't have any glue on how we could make it work

**Comment by Chriztiaan** [comment]
Has this been resolved?

**Comment by udomnohara** [comment]
any progress?
